src/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useCallback} from 'react'
+import type {ComponentMeta, ComponentStory} from '@storybook/react'
+import {View, Button, Text, StyleSheet} from 'react-native'
+import Modal from './Modal'
+
+export default {
+  title: 'components/Modal',
+  component: Modal,
+} as ComponentMeta<typeof Modal>
+
+export const ModalComponent: ComponentStory<typeof Modal> = () => {
+  const [visible, setVisible] = React.useState(false)
+
+  const showModal = useCallback(() => setVisible(true), [])
+  const hideModal = useCallback(() => setVisible(false), [])
+
+  return (
+    <View style={styles.viewContainer}>
+      <Modal visible={visible} onClose={hideModal} contentContainerStyle={styles.contentContainerStyle}>
+        <View>
```

Comment:
Can we use a props {children} in modal ?

Reply:
This is just a storyboard file for testing. Can check the Modal.tsx file